layouts/explore-layout.vue

```diff
@@ -31,11 +31,18 @@
         </div>
       </main>
     </div>
-    <Buy />
     <template v-if="listingCartEnabled">
       <ListingCartMini />
       <ListingCartModal />
     </template>
+
+    <Buy @success="handleBuy" />
+
+    <MessageNotify
+      v-if="message"
+      :title="message.title"
+      :subtitle="message.subtitle"
+      @close="message = null" />
```

Comment:
Do we really need to put some logic in layout? it feels bad for me

any other option? (maybe put it in another component)